CoreFoundation/Base.subproj/SymbolAliases

```diff
@@ -1,105 +0,0 @@
-
-
-# This is specific to Swift open source; Foundation's NSCFConstantString is used as our constant string class reference
-__TMC15SwiftFoundation19_NSCFConstantString ___CFConstantStringClassReference
```

Comment:
These aliases are needed to properly represent the symbols. Why were they removed?